src/server/repository/assessment/section/getMany.ts

```diff
@@ -37,16 +37,21 @@ export const getMany = async (
     `,
     [cycle.uuid],
     ({ data }: { data: Array<Omit<Section, 'subSections'> & { sub_sections: Array<SubSection> }> }) => {
-      // eslint-disable-next-line camelcase
-      return data.map(({ sub_sections, props: { labels, ...props }, ...section }) => ({
-        ...Objects.camelize(section),
-        props: { ...Objects.camelize(props), labels },
-        // eslint-disable-next-line camelcase
-        subSections: sub_sections.map(({ props: { anchors, labels, ...props }, ...subSection }) => ({
-          ...Objects.camelize(subSection),
-          props: { ...Objects.camelize(props), anchors, labels },
-        })),
-      }))
+      return (
+        data
+          // eslint-disable-next-line camelcase
+          .filter((d) => d.sub_sections?.length > 0)
```

Comment:
Instead of filtering , can the condition be added in the where condition of the query above ?